### PR TITLE
Create PRINCIPLES.md

### PR DESCRIPTION
**Pull request by JEM-Mosig**
**Proposed changes**:
- Add the 'Principles for Building Ethical Conversational Assistants' document

### CODE REVIEW

File: PRINCIPLES.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Principles for Building Ethical Conversational Assistants
```
Comment (lunelson) [inline PRINCIPLES.md:1]: We don't need the `h1` tag in the document, because it is specified on the website side

Comment (JEM-Mosig) [reply PRINCIPLES.md:1]: So I delete that line?

Comment (lunelson) [reply PRINCIPLES.md:1]: Yes. I've added that title to the configuration in the website that will pull in the file

**Comment by lunelson** [comment]
Closes https://github.com/RasaHQ/rasa-website/issues/761

**Comment by lunelson** [comment]
@m-vdb this PR only contains one file, I'd like to go ahead and merge it even though there are failed checks, since they don't seem relevant to this change at all, wdyt?

**Pull request merged**